boards/arm/frdm_k64f/doc/index.rst

```diff
@@ -321,6 +321,15 @@ should see the following message in the terminal:
    ***** Booting Zephyr OS v1.14.0-rc1 *****
    Hello World! frdm_k64f
 
+**note**
```

Comment:
This should be documented in the test, not in the board documentation.

Reply:
Have updated to the test, can help check whether it is correct？